client/src/services/contactQuestioning.service.ts

```diff
@@ -0,0 +1,42 @@
+
+import { Subject } from 'rxjs';
```

Comment:
I don't think its the right place to put these things, it's better that you'll spread them to models/use files
it can work but it's very important that you'll continue with the right convention

Reply:
But it is not a model! it's a service! you didn't write services in the app yet.
You can see in this link (the link that Tair send us) where the service is located
![image](https://user-images.githubusercontent.com/85481012/128816713-b98c6a53-def0-46de-a054-0511cd0125b6.png)

https://jasonwatmore.com/post/2019/02/13/react-rxjs-communicating-between-components-with-observable-subject